bokeh/models/axes.py

```diff
@@ -146,6 +146,16 @@ class Axis(GuideRenderer):
     main plot area.
     """)
 
+    fixed_location = Either(String, Float, help="""
```

Comment:
`String`?

Reply:
Also, there should be an explicit default given that the default in bokehjs is `null`.

Reply:
String is for categorical coordinates. It should probably accept various kinds of tuples for nested categorical support, but OK to start with this.